quipucords/tests/api/source/test_source.py

```diff
@@ -1250,7 +1251,7 @@ def test_negative_create_non_network_with_excluded_hosts(
     @pytest.mark.parametrize(
         "source_type", (ds for ds in DataSources if ds != DataSources.NETWORK)
     )
-    @pytest.mark.parametrize("hosts_range", ("1.2.3.4/5", "1.2.3.[0:255]"))
+    @pytest.mark.parametrize("hosts_range", ("1.2.3.4/30", "1.2.3.[0:255]"))
```

Comment:
The CIDR range "1.2.3.4/30" was causing these tests to time out. In the previous implementation, we converted CIDR ranges to Ansible ranges, but now we expand both. The given range includes 8,388,608 IP addresses, so I replaced it with a smaller one. As a team, we agreed to address potential performance issues caused by CIDR expansion in a separate story. Given this, I considered the change acceptable, as it does not impact the test’s evaluation.